integration-tests/tests/link-to-new-workspace.js

```diff
@@ -15,19 +15,18 @@ const testLinkToNewWorkspaceFn = withUserToken(async ({ testUrl, page, token })
   const newWorkspaceBillingAccount = 'general-dev-billing-account'
   await click(page, clickable({ textContains: 'Start with a new workspace' }))
   await fillIn(page, '//*[@placeholder="Enter a name"]', `${newWorkspaceName}`)
-  await click(page, clickable({ text: 'Select a billing project' }))
-  await click(page, clickable({ text: `${newWorkspaceBillingAccount}` }))
+  await select(page, 'Billing project', `${newWorkspaceBillingAccount}`)
   await noSpinnersAfter(page, { action: () => click(page, clickable({ textContains: 'Create Workspace' })) })
   try {
     // Wait for bucket access to avoid sporadic failures
     await checkBucketAccess(page, newWorkspaceBillingAccount, newWorkspaceName)
-    await page.url().includes(newWorkspaceName)
+    await findText(page, `${newWorkspaceBillingAccount}/${newWorkspaceName}`)
```

Comment:
@samanehsan FYI, removing the page.url call, which won't actually fail the test even if it returns `false`, with looking for the expected path in the top context bar.

Reply:
Thanks!